### ISSUE TITLE
Variable "database_error" does not exist.

### ISSUE THREAD

**Issue by binarious** [contao/installation-bundle#39]
`/contao/install` throws the following error when the database doesn't exist:
```
Variable "database_error" does not exist.
in vendor/contao/installation-bundle/src/Resources/views/database.html.twig at line 7
```

Steps to reproduce:
- `composer create-project contao/standard-edition contao-test`
- Enter invalid db settings
- Start webserver
- Navigate to https://youraddress/install.php and accept the license
- Enter password and submit

Environment:
- macOS 10.12.2
- PHP 7.0.14 (via fpm-fcgi)
- webserver: Caddy 0.9.4
- Composer 1.3.0

**Comment by leofeyer** [comment]
Here's how you can fix it until the release of version 1.2.2:

https://github.com/contao/installation-bundle/commit/5f5db007a0fa7784d9456835bbd6bfce39019965#diff-e7a6ffda6efa8c6996042c3f045a8caeR7

**Comment by leofeyer** [comment]
Fixed in b7988263a51f7b469630ded2bbad15419e51c4c4.
